microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -243,6 +243,11 @@ class AzureImageStandard(TestSuite):
         ),
         # pam_unix,pam_faillock
         re.compile(r"^(.*pam_unix,pam_faillock.*)$", re.M),
+        # benign warning from mellanox on freebsd
```

Comment:
typo, and add some details, why it can be ignored.

Reply:
Ok I will update the comment.